hydrator.py

```diff
@@ -1,30 +1,44 @@
 import datetime
 import time
-
+import PySimpleGUI as sg
 from playsound import playsound
 
+sg.theme('Topanga')
 
-def notify():
-    print(f"{datetime.datetime.now().strftime('%Y/%m/%d %H:%m')}: Time to drink")
-    play_notification()
-
-
-def play_notification():
-    playsound("./ienba_notification.wav")
+now = f"""
 
+{datetime.datetime.now().strftime('%Y/%m/%d %H:%m')}: Time to drink water...      
+    
 
-print("""
+"""
+startup_message = """
     Hydrator Copyright (C) 2022 Binula Kavisinghe, Tarith Jayasooriya
     This program comes with ABSOLUTELY NO WARRANTY.
     This is free software, and you are welcome to redistribute it
     under certain conditions;
     
     This software uses "Notification" by IENB ( https://freesound.org/people/IENBA/sounds/545495/ ) from freesound licensed under CCBY3.0
 
-    You will hear a tone in every hour
-""")
-play_notification()
+    You will hear a tone and see a popup in every hour.
```

Comment:
remove "in", "see a popup in every hour" sounds wrong maybe "see a popup every hour?"

Reply:
noted